### DISCUSSION TITLE
Migrate to ultrafunkamsterdam/nodriver

### DISCUSSION THREAD

**Discussion by blu3b3rry-sh** [FlareSolverr/FlareSolverr#1126] [Ideas]
Hi, [ultrafunkamsterdam/nodriver](https://github.com/ultrafunkamsterdam/nodriver) is the official successor of [ultrafunkamsterdam/undetected-chromedriver](https://github.com/ultrafunkamsterdam/undetected-chromedriver/), and has following advantages :
- no chromedriver binary or Selenium dependency
- performance increase
- less detections
- fully asynchronous
- "Up and running in 1 line of code*"

Do you plan to migrate to [ultrafunkamsterdam/nodriver](https://github.com/ultrafunkamsterdam/nodriver) once it will be released (currently release candidate) ?

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/discussions/1096

**Reply by blu3b3rry-sh** [reply]
Sorry for this duplicate 😅
